packages/mobile/src/components/composer-input/ComposerInput.tsx

```diff
@@ -126,6 +124,16 @@ export const ComposerInput = forwardRef(function ComposerInput(
   const lastKeyPressMsRef = useRef<number | null>(null)
   const { data: track } = useGetTrackById({ id: entityId ?? -1 })
 
+  useEffect(() => {
+    setUserMentions(presetUserMentions.map((mention) => `@${mention.handle}`))
```

Comment:
do we need state for this stuff or could we just compute it each time?

Reply:
i think we need bc this is also the state to update when the user autocompletes